distributed/deploy/ssh.py

```diff
@@ -45,6 +45,9 @@ async def close(self):
     def __repr__(self):
         return f"<SSH {type(self).__name__}: status={self.status}>"
 
+    def _repr_html_(self):
+        return super()._repr_html_()
```

Comment:
Sorry I meant to delete this whole method. It is redundant.

Reply:
Oh! I am surprised it still works after removing this.
Does it know which `_repr_html_`  to call from the fact that Worker is a child class of Process?

Reply:
Yes, python has a strict method resolution procedure, which class inheritance relies on. You can see the places python will look to find an attribute with, for instance, `Worker.mro()` (this is the class object, not an instance).

Reply:
Yeah. This is all part of class inheritance. Unless you state otherwise it takes on all the methods of the parent.

Reply:
This `_repr_html_` method still needs removing.

Reply:
Oh yes! Removed.
I allso removed the __repr__ too and edited the ProcessInterface's __Repr__ to follow the same pattern.

Reply:
@jacobtomlinson Wow, I am super tired today :smile: I had commited the changed, but forgot to push.